### PR TITLE
Add xblock-sdk workbench support & basic scenarios

### PR DESCRIPTION
**Pull request by z4y4ts**

### CODE REVIEW

File: video_xblock/video_xblock.py [hunk 1]
```diff
@@ -296,6 +304,7 @@ def srt_to_vtt(self, request, _suffix=''):
         return Response(self.convert_caps_to_vtt(caps))
 
 
+@XBlock.wants('modulestore')
```
Comment (polesye) [inline video_xblock/video_xblock.py]: Why don't you use `needs` here?

File: video_xblock/workbench/mixin.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    """
+    WorkbenchMixin adds XBlock-SDK workbench support to an XBlock.
+    """
+
+    @staticmethod
+    def workbench_scenarios():
+        """
+        Load scenarios for display in the workbench from xml files.
+        """
+        return loader.load_scenarios_from_path('workbench/scenarios')
```
Comment (polesye) [inline video_xblock/workbench/mixin.py]: Another way, just use lxml to dynamically generate xml

File: video_xblock/mixins.py [hunk 3]
```diff
@@ -54,3 +54,44 @@ def populate_default_values(self, fields_dict):
         for key, value in self.settings.items():
             fields_dict.setdefault(key, value)
         return fields_dict
+
+
+class LocationMixin(XBlock):
```
Comment (polesye) [inline video_xblock/mixins.py]: if we need it only in test purposes, can we simply decorate VideoXblock instance and add these methods?
Something like that:
```py
instance = LocationDecorator(VideoXBlock(...))
```
Does it make sense?

File: video_xblock/tests/test_mixins.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+from video_xblock.tests.base import VideoXBlockTestBase
+from video_xblock.utils import loader
+
+
+class ContentStoreMixinTest(VideoXBlockTestBase):
+    """Test ContentStoreMixin"""
+
+    @patch('video_xblock.video_xblock.import_module')
+    def test_import_from(self, import_module_mock):
```
Comment (polesye) [inline video_xblock/tests/test_mixins.py]: docstrings?

File: video_xblock/video_xblock.py [hunk 5]
```diff
@@ -43,6 +42,13 @@ class ContentStoreMixin(XBlock):
     from `xmodule.contentstore`
     """
 
+    @staticmethod
+    def import_from(module, klass):
```
Comment (polesye) [inline video_xblock/video_xblock.py]: does it make sense to have it as a static method? It looks like an independant utility.

File: video_xblock/video_xblock.py [hunk 6]
```diff
@@ -362,18 +371,30 @@ class PlaybackStateMixin(XBlock):
...
+        """
+        Utility method returns course's language.
+
+        Falls back to 'en' if runtime doen't provide `modulestore` service.
+        """
+        if self.runtime.service(self, 'modulestore'):
+            course = self.runtime.modulestore.get_course(self.course_id)
+            return course.language
+
+        return 'en'
```
Comment (polesye) [inline video_xblock/video_xblock.py]: some cont DEFAULT_LANGUAGE?

File: video_xblock/video_xblock.py (hunk 5 above)
Comment (z4y4ts) [inline video_xblock/video_xblock.py]: Good point

File: video_xblock/video_xblock.py (hunk 1 above)
Comment (z4y4ts) [inline video_xblock/video_xblock.py]: Yeah, good point. This module is crucial for normal flow.

File: video_xblock/workbench/mixin.py (hunk 2 above)
Comment (z4y4ts) [inline video_xblock/workbench/mixin.py]: I don't feel like adding another level of complexity.
Declarative nature of xml scenarios fits well here.

File: video_xblock/mixins.py (hunk 3 above)
Comment (z4y4ts) [inline video_xblock/mixins.py]: That's not only for tests, but to simplify main VideoXBlock logic a bit.

File: video_xblock/tests/test_mixins.py [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+            self.assertEqual(self.xblock.static_content, 'StaticContent_test')
+            service_mock.assert_called_once_with(self.xblock, 'contentstore')
+            sc_mock.assert_called_once()
+
+
+class LocationMixinTests(VideoXBlockTestBase):
+    """Test LocationMixin"""
+
+    def test_xblod_doesnt_have_location_by_default(self):
```
Comment (TamoshaytisV) [inline video_xblock/tests/test_mixins.py]: typo - `test_xblock_...`

Comment (z4y4ts) [inline video_xblock/tests/test_mixins.py]: Nice catch! Thanks for pointing out!
Will be fixed in next PR

**Pull request merged**